test/integration/simple_example/controls/jenkins.rb

```diff
@@ -33,7 +33,7 @@
       end
 
       it "sets the launch zone of workers to the correct zone" do
-        expect(config.at_xpath('//clouds/*/configurations/*/zone').children[0].text).to end_with "/us-east4-a"
+        expect(config.at_xpath('//clouds/*/configurations/*/zone').children[0].text).to end_with "/us-east4-c"
```

Comment:
Add variable `jenkins_workers_zone` to the `inspec.yml` to use it for the comparison.